typed-racket-doc/typed-racket/scribblings/guide/caveats.scrbl

```diff
@@ -244,7 +244,17 @@ anticipate with their pattern matching, e.g.:
   (eval:error
    (size 42))]
 
+Patterns involving an ellipsis @racket[...] for repetition may generate a 
+@racket[for] loop that requires annotations on variables to type check. 
+The (deliberately obscure) code below does not type check without the type 
+annotation on the match pattern variable @racket[c].
 
+@examples[#:label #f #:eval the-eval
+  (: without-10 : (Listof Integer) -> (Listof Integer))
```

Comment:
The name `without-10` is kinda weird, now that the first element is not 10.

Reply:
Name improved to `skip-a-num`, which is kind of casual.
I also consided `rest-nums` and variations on it (`rest-nums-only`, `rest-numeric`, `rest-intonly`).
I don't have a real preference or guiding principle for the choice.